src/runtime/manifest-primitive-parser/smoke/toy.html

```diff
@@ -0,0 +1,161 @@
+<style>
+  body {
+    height: 100vh;
+    font-family: sans-serif;
+    display: flex;
+    flex-direction: column;
+  }
+  body, pre {
+    margin: 0;
+    box-sizing: border-box;
+  }
+  pre {
+    color: navy;
+    background-color: #FAFAFA;
+  }
+  [columns] {
+    display: flex;
+    align-items: stretch;
+    height: 80vh;
+  }
+  [column] {
+    flex: 1;
+    border: 1px dotted silver;
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+  }
+  [flex] {
+    flex: 1;
+  }
+  #logElt {
+    padding: 12px;
+  }
+  textarea {
+    flex: 1;
+    padding: 4px;
+    border: none;
+    outline: none;
+  }
+</style>
+
+<div columns>
+  <div column>
+    <textarea id="inputElt" spellcheck="false"></textarea>
+  </div>
+  <div column id="treeElt">
+    <pre id="outputElt"></pre>
+  </div>
+</div>
+<pre flex id="logElt"></pre>
+
+<script type="module">
+  import {Loader} from '../../../../build/platform/loader.js'
+  import {ManifestPrimitiveParser} from '../../../../build/runtime/manifest-primitive-parser/manifest-primitive-parser.js';
+
+  const dom = (tag, props, parent) => parent.appendChild(Object.assign(document.createElement(tag), props));
+  const {inputElt, logElt, outputElt} = window;
+
+  const loader = new Loader();
+
+  inputElt.oninput = async () => {
+    try {
+      logElt.innerText = '';
+      const content = inputElt.value;
+      const opts = {
+        loader,
+        filename: `http://localhost/projects/arcs/arcs/particles/Bespoke/Bespoke.arcs`
+      };
+      const ast = await ManifestPrimitiveParser.parse(content, opts);
+      outputElt.innerText = JSON.stringify(ast, (k,v) => k === 'location' ? undefined : v, '  ');
+      //
+      const kinds = new Set();
+      ast.forEach(item => kinds.add(item.kind));
+      logElt.innerText = [
+        `Ast Kinds: ${[...kinds]}`,
+        `Imports: ${ManifestPrimitiveParser.extract('import', ast).length}`,
+        `Recipes: ${ManifestPrimitiveParser.extract('recipe', ast).length}`,
+        `Particles: ${ManifestPrimitiveParser.extract('particle', ast).length}`,
+        `Stores: ${ManifestPrimitiveParser.extract('store', ast).length}`
+      ].join('\n');
+    } catch(x) {
+      logElt.innerText = x.toString();
+      console.error(x);
+    }
+  };
+
+  const sample = `// Overview Action Example.
+// From: https://docs.google.com/presentation/d/1i-mTjlyWFSHYxvcJj98fIkUkHSu7ya3COa2wZeiqyi8/edit#slide=id.g5907ddee85_0_75
+
+import '../Dataflow/Datastores.arcs'
```

Comment:
I had to comment this out to make the sample parse correctly.

Reply:
See note about paths above.